2017/A10-Insufficient_Logging%2526Monitoring.md

```diff
@@ -40,7 +42,7 @@ Insufficient logging, detection, monitoring and active response occurs any time:
 * Penetration testing and scans by [DAST](/www-community/Vulnerability_Scanning_Tools) tools (such as [OWASP ZAP](/www-project-zap)) do not trigger alerts.<br>
 * The application is unable to detect, escalate, or alert for active attacks in real time or near real time.<br>
 
-You are vulnerable to information leakage if you make logging and alerting events visible to a user or an attacker (see [A3:2017-Sensitive Data Exposure](Top_10-2017_A3-Sensitive_Data_Exposure)).
+You are vulnerable to information leakage if you make logging and alerting events visible to a user or an attacker (see [A3:2017-Sensitive Data Exposure](A3-Sensitive_Data_Exposure.md)).
```

Comment:
Don't use any extension just `A3-Sensitive_Data_Exposure` should be fine....same throughout.